addons/mail/static/src/discuss/core/common/attachment_panel.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.AttachmentPanel">
+        <t t-set="title">Attachments</t>
+        <ActionPanel title="title">
+            <div class="flex-grow-1" t-att-class="{
+                'd-flex justify-content-center align-items-center': props.thread.attachments.length === 0,
+            }">
+                <p t-if="props.thread.attachments.length === 0" class="text-center fst-italic text-500 fs-6">
+                    <t t-if="props.thread.type === 'channel'">This channel doesn't have any attachments.</t>
+                    <t t-else="">This conversation doesn't have any attachments.</t>
```

Comment:
Note that `AttachmentPanel` only used with channels. The `t-else` is currently unreachable

Reply:
It is reachable when in a 1-to-1 chat

Reply:
Oops, misread `.model === 'discuss.channel'`. Then it's ok